apps/bare-expo/ios/Podfile.lock

```diff
@@ -3876,14 +3876,14 @@ SPEC CHECKSUMS:
   ReactAppDependencyProvider: c55bd3998b0b548d2e81f5aae4758096d4321a7a
   ReactCodegen: c63ed13899d192e3ac78b794f4149e950dc86122
   ReactCommon: 558d37855c8b281c4fdfe8f9ba2d7bc102fb55ff
-  ReactNativeDependencies: 0e21bdd14e5112c3663bdfedc68157a52b80c310
+  ReactNativeDependencies: 6012c41b8b2962e2af6d9cb299c05c1811be9d87
```

Comment:
good to `rm -rf Pods` and run pod install again. not sure if that change is because of outdated pod installation